source/includes/usage-examples/code-snippets/count.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri = "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
+
+func main() {
+
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(context.TODO()); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin countDocuments
+	coll := client.Database("sample_mflix").Collection("movies")
+	filter := bson.D{{"countries", "Spain"}}
+
+	collCount, err := coll.EstimatedDocumentCount(context.TODO())
+	count, err := coll.CountDocuments(context.TODO(), filter)
```

Comment:
I believe `err` will be shadowed here by the second assignment. Suggest using different names for the vars

Reply:
```suggestion
	collCount, collCountErr := coll.EstimatedDocumentCount(context.TODO())
	count, countErr := coll.CountDocuments(context.TODO(), filter)
```